packages/host/app/routes/card.gts

```diff
@@ -55,6 +71,12 @@ export default class RenderCard extends Route<Model | null> {
       }
 
       if (operatorModeEnabled) {
+        if (params.authMode) {
+          this.matrixService.setAuthMode(params.authMode);
```

Comment:
Is it possible to figure out auth mode from given params? for example if `sid` and `clientSecret` are present then show the reset password form?

Reply:
Currently we use `sid` and `clientSecret` params in reset password scenario only, so yeah we can figure out the auth mode from those params.